MekHQ/src/mekhq/gui/utilities/MarkdownRenderer.java

```diff
@@ -56,6 +56,18 @@ public static String getRenderedHtml(String input) {
         if(null == input) {
             return "";
         }
+        /*
+         * Tabs will cause problems here because Markdown thinks
+         * of them as block quotes. However, many users will probably
+         * use them to identify beginning of paragraphs. So replace all
+         * tabs with line returns. Ultimately, if we ever get up a markdown
+         * editor we should also check for tabs there and correct them in the 
+         * input
+         */
+        input = input.replaceAll("\\t+", "\n\n");
+        //This is also going to happen if indentation is used with spaces
+        input = input.replaceAll("\\s\\s\\s\\s+", "\n\n");
+        
```

Comment:
https://github.com/atlassian/commonmark-java/blob/bf8bea9dfcb49ee43e20372b42b43727d9a15da2/commonmark-test-util/src/main/resources/spec.txt#L340

This spec seems to say these are treated as indentation rather than blockquotes (a la `>`). I think it would be a good idea to leave this out and see what sort of "problems" show up in folks files.

Reply:
Or, you could remove the `IndentedCodeBlock` from the set given to the parser builder:

https://static.javadoc.io/com.atlassian.commonmark/commonmark/0.13.0/org/commonmark/parser/Parser.Builder.html#enabledBlockTypes-java.util.Set-

Reply:
As far as what happens is that it basically breaks the rendering. The text becomes much smaller and fails to wrap.

The issue is users who use indentation to begin new paragraphs without an empty line in between, which is still a common way to write. If we do nothing, the rendering will make it go haywire. If we removed the IndentedCodeBlock then I think all of their paragraphs will run together. I will test it to make sure. I have tested the current solution and it may not be elegant but it definitely works.

Reply:
Ok, in b7b10d5ce, I changed the code to remove the regex substitions and use the enableBlockTypes method in the parser builder to remove potentially problematic block types.

The one down side of this approach is that users who use indentation to write paragraphs will have everything smooshed into one giant paragraph, but they will probably figure it out quickly enough. We can keep an eye on what users say.